deploy/crds/certmanager.k8s.io_certificates_crd.yaml

```diff
@@ -0,0 +1,252 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
@rashmi43 Why we need to add this certmanager CRD into the IAM operator manifest?

Reply:
we do not need it, it somehow got added, I have removed it, its not there in any other version